siphon/simplewebservice/wyoming.py

```diff
@@ -99,13 +108,31 @@ def _get_data_raw(self, time, site_id, region='naconf'):
         site_id : str
             Site id for which data should be downloaded
         region : str
-            The region in which the station resides. Defaults to `naconf`.
+            Region for the sounding data. Valid regions are `North America`,
+            `South America`, `South Pacific`, `New Zealand`, `Antarctica`,
+            `Arctic`, `Europe`, `Africa`, `Southeast Asia`, `Mideast`. Defaults
+            to `North America`.
 
         Returns
         -------
         a file-like object from which to read the data
 
         """
+        # Translate the human readable region to the region abbreviation
+        regions = {'north america': 'naconf', 'south america': 'samer',
+                   'south pacific': 'pac', 'new zealand': 'nz',
+                   'antarctica': 'ant', 'arctic': 'np', 'europe': 'europe',
+                   'africa': 'africa', 'southeast Asia': 'seasia',
+                   'mideast': 'mideast'}
+
+        try:
+            region = regions[region.lower()]
+        except KeyError:
+            raise KeyError("Invalid region {}. Valid regions are 'North America'"
```

Comment:
should probably raise a `ValueError` instead of `KeyError`, since the error is that the user passed in a bad value.